etc/gen_all_secrets

```diff
@@ -38,7 +38,8 @@ while getopts ':q' OPT ; do
 done
 shift $((OPTIND-1))
 
-create_pw_file gendbpasswords dbpasswords.secret
-create_pw_file genrestapicredentials restapi.secret
-create_pw_file gensymfonysecret symfony_app.secret
-create_pw_file genadminpassword initial_admin_password.secret
+cd "$(dirname "$0")"
```

Comment:
I would propose to only changing this single line. I think that should be enough to make the output go into the `etc` directory (specifically: the directory this script resides in). It also preserves script name in the message on line 13, which becomes more ambiguous with a `./` prefixed.

Reply:
I think that would work if line 16 was also changed to `"./$SCRIPT" > "$FILE"`, because `$0` can be a relative path so is unreliable after `cd`.

Reply:
Indeed, you're right.